mailer.js

```diff
@@ -390,12 +434,17 @@ module.exports = function (log) {
   Mailer.prototype.postVerifyEmail = function (message) {
     log.trace({ op: 'mailer.postVerifyEmail', email: message.email, uid: message.uid })
 
+    var templateName = 'postVerifyEmail'
     // special utm params, just for this email
     // details at github.com/mozilla/fxa-auth-mailer/issues/110
-    var postVerifyUtmParams = '?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified'
```

Comment:
@rfk Looks like previously we used `fx` as the prefix, should we reuse this to keep consistency or rename to `fxa`? Currently, you have the ability to override the params when building links so we could keep them different, but would prefer them the same.